services/backup/backup_service.go

```diff
@@ -226,6 +289,58 @@ func (s *Service) RestoreBackup(ctx context.Context, serviceID, artifactID strin
 	return restoreID, nil
 }
 
+// SwitchMongoPITR switches Point-in-Time recovery feature for mongoDB with given serviceID.
+func (s *Service) SwitchMongoPITR(ctx context.Context, serviceID string, enabled bool) error {
```

Comment:
is this func idempotent?

Reply:
Nope, it switches PITR option on the target service node.